harvester/cli.py

```diff
@@ -31,10 +34,91 @@ def main(ctx: click.Context, verbose: bool) -> None:
 
 @main.command()
 @click.pass_context
-def shell(ctx: click.Context) -> None:
-    """Run a bash shell inside the docker container."""
-    # ruff: noqa: S605, S607
-    os.system("bash")
+def docker_shell(ctx: click.Context) -> None:
```

Comment:
Hmm, if the Click command is not using an object from the Click context or passing an object to the Click context, is `@click.pass_context` still needed? 🤔

Reply:
Nice catch!  Removing from all CLI commands that don't need it.  This _also_ surfaced a bug I hadn't caught when refactoring the Makefile commands (need to call CLI command `docker-shell` not `shell`).